vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -159,6 +159,7 @@ type internal FSharpWorkspaceServiceFactory [<Composition.ImportingConstructor>]
                                         nameof enableLiveBuffers, enableLiveBuffers
                                         nameof useSyntaxTreeCache, useSyntaxTreeCache
                                         nameof enableParallelReferenceResolution, enableParallelReferenceResolution
+                                        nameof enableInMemoryCrossProjectReferences, enableInMemoryCrossProjectReferences
```

Comment:
Telemetry had some changes in the meantime - this feels like the correct way to merge them

Reply:
Sorry, wasn't too elaborate yesterday, I meant - is there some schema for telemetry data somewhere that needs to be modified accordingly? Also, just to be sure - do we even want this in telemetry? cc: @vzarytovskii

Reply:
That's fine, it can be modified once data flows in. It will show in our gdpr portal.